pkg/domain/domain.go

```diff
@@ -1944,6 +1944,42 @@ func decodePrivilegeEvent(resp clientv3.WatchResponse) PrivilegeEvent {
 	return msg
 }
 
+func ignoreBatchData(ch clientv3.WatchChan) {
+	ticker := time.NewTicker(5 * time.Millisecond)
+	defer ticker.Stop()
+	for i := 0; i < 1024; i++ {
+		select {
+		case <-ch:
+		case <-ticker.C:
+			return
+		}
+	}
+}
+
+func batchReadMoreData(ch clientv3.WatchChan, event PrivilegeEvent) PrivilegeEvent {
+	ticker := time.NewTicker(5 * time.Millisecond)
+	defer ticker.Stop()
+	for i := 0; i < 128; i++ {
```

Comment:
Why the batch size is different with previous one? Is there any reason for this?

Reply:
When loadAll happen, we can ignore all the loadSomeUsers event, because after loadAll, load one specifiy user is a redundant operation.

But I still limit a 1024 because imagine this case: one privilege event come, 4.99ms later, another privilege event come ... then 4.99ms later, yet another privilege event come ...
If there is no limit, in this extreme case, the `loadAll` will never be called, because we're batching forever.

128 make the event more real-time, while 1024 discard as much events as possible but avoid the extreme cases.